ancpbids/plugins/plugin_dsvalidator.py

```diff
@@ -48,7 +53,7 @@ def execute(self, dataset, report: ValidationPlugin.ValidationReport):
             for ref in entity_refs:
                 if ref.key not in entities:
                     report.error(
-                        "Invalid entity '%s' in artifact '%s'" % (ref.key, artifact.get_relative_path()))
+                        "Invalid entity '%s' in artifact '%s'" % (ref.key, artifact.get_relative_path()), ref.key)
```

Comment:
ref.key is a string, better to use offender=artifact, then we have a node from the graph to traverse up/down